src/helpers/getCurrenciesValues.ts

```diff
@@ -0,0 +1,11 @@
+import { Currencies } from '../types/CountriesTypes';
+
+export const getCurrenciesValues = (currencies:Currencies) => {
+  const currenciesKeys = Object.keys(currencies || {});
+  
+  const currenciesValues = currenciesKeys.map(key => {
+    return currencies ? currencies[key as keyof Currencies] : { name: "", symbol: "" };
```

Comment:
Siento que más bien la validación pudiera ser en plan:

```suggestion
    return currencies[key as keyof Currencies] || { name: "", symbol: "" };
```

Ya que los currencies va a venir como un objeto con una propiedad o vacío en todo caso según los types. 🐱